.github/workflows/pr-title-check.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - name: Validate PR Title
         run: |
-          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major|docs|ci)\(([a-z_-]+)\):(.+|\d+)$ ]]; then
+          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major|docs|ci)\(([a-z_-]+)\)!*:(.+|\d+)$ ]]; then
```

Comment:
pq incluimos uma exclamação?

Reply:
Because if the change is Major, the PR title needs to be similar to this:

`[fix(tokens)!: fix all letter spacing values](https://github.com/juntossomosmais/atomium/pull/537#top)`